app/models/manageiq/providers/ibm_cloud/vpc/manager_mixin.rb

```diff
@@ -16,7 +16,11 @@ def connect(options = {})
     key = authentication_key(options[:auth_type])
     region = options[:provider_region] || provider_region
     sdk = self.class.raw_connect(key)
-    sdk.vpc(:region => region)
+    if options[:service] == 'ActivityTrackerApiV1'
```

Comment:
since we're calling the service `"events"` belong (as in `sdk.events`) how about simply `"events"` as the service name here?

Reply:
sure, that makes more sense